docs/contributor-guide/developer-guide/system-design/data-intake.md

```diff
@@ -0,0 +1,47 @@
+# Data intake
+
+This page describes where various pieces of data related to user accounts originate.
+
+CiviForm's use of a flexible, admin-defined [backend data model](./backend-data-model.md) complicates investigations into where a given piece of data about an applicant comes from.
+
+## `ApplicantData`
+
+Generally speaking, data stored in [ApplicantData](https://github.com/civiform/civiform/blob/main/server/app/services/applicant/ApplicantData.java) (per the [backend data model](./backend-data-model.md) schema) originates from applicants (or TIs acting on an applicant's behalf) answering questions.
+
+### Hard-coded ApplicantData entries
+
+#### Well known paths
+
+["Well known paths"](https://github.com/civiform/civiform/blob/main/server/app/services/WellKnownPaths.java#L4) are hard-coded pointers into `ApplicantData` that enable hard-coded logic for reading and writing specific pieces of data in the flexible schema. These were added to support features such as searching for an applicant by their name or date of birth and displaying the applicant's name in the UI:
+- TIs can directly update applicant name and DoB ([example](https://github.com/civiform/civiform/blob/main/server/app/services/ti/TrustedIntermediaryService.java#L199-L200))
+- Authentication logic sets the applicant's name using the identity provider-furnished value when they login ([example](https://github.com/civiform/civiform/blob/main/server/app/auth/oidc/applicant/ApplicantProfileCreator.java#L113))
+
+#### Preferred locale
+
+The applicant's preferred locale is stored in `ApplicantData` with [hard coded read and write logic](https://github.com/civiform/civiform/blob/main/server/app/controllers/applicant/ApplicantInformationController.java#L179) to support the applicant specifying it.
+
+## Account data
+
+Data stored on an applicant's account originates from identity providers as well as CiviForm Admin and Trusted Intermediary user behavior.
+
+### Email address
+
+Each account's email address originates either from the identity provider the account authenticated with or is specified by the TI when they create a new client account. It is unset for guest accounts.
```

Comment:
The TIs can create a client without email address too (Nick added this feature). Empty email, can be guest or Clients of TIs without email address

Reply:
reworded